public/css/main.css

```diff
@@ -1005,4 +1005,13 @@ body.darkBG .keynav-target {
 }
 .fs15 span.rm-spacing-tb .int {
   line-height: 1.6rem;
+}
+.sync-progress-bar {
+  animation: glowGreen 3000ms infinite;
+}
+.sync-progress-bar span {
+  vertical-align: -webkit-baseline-middle;
+}
+.container div.sync-progress {
```

Comment:
If we reduce/limit the width of the progress bar itself, we might want to center the sync-progress div with `margin: auto`.